prog.go

```diff
@@ -613,6 +613,24 @@ func (p *Program) ID() (ProgramID, error) {
 	return ProgramID(info.id), nil
 }
 
+// ProgStats holds statistics of a program.
+type ProgStats struct {
+	RunTimeNs uint64
+	RunCnt    uint64
+}
+
+// Stats returns measured statistics of a the program.
+func (p *Program) Stats() (ProgStats, error) {
```

Comment:
This should be on ProgramInfo, with a getter similar to `ProgramInfo.ID()`. Also document that this returns zero (?) values without EnableStats.

Reply:
I decided against `func (i *ProgramInfo) Stats() (ProgramStats, error) ` as it requires more syscalls to get the same information.
E.g. one has to get the FD from `ProgramInfo` first before `bpfGetProgInfoByFD` can be used. And storing FD in `ProgramInfo` feels not correct. what do you think?

Reply:
Take a look at newProgramInfoFromFd, which is called by Program.Info. There you can pull out the necessary info for ProgramStats and store it in an unexported field in ProgramInfo. The getter then allows the user to retrieve the `ProgramStats` while also telling them whether the feature is available in the first place (on old kernels we can't populate the info). All of this is still only a single syscall.